packages/ui/src/components/Text/index.tsx

```diff
@@ -159,7 +166,7 @@ export const Text = ({
         as={as}
         placement={placement}
         prominence={prominence}
-        color={color}
+        color={computedSentiment}
```

Comment:
Should we rename the prop too?

Reply:
It's internal to the component but I can check indeed